notebooks/official/feature_store/online_feature_serving_fetching_multiple_entities.ipynb

```diff
@@ -0,0 +1,850 @@
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "d0032be9c2d6"
+      },
+      "source": [
+        "Let's try reading 1000 entities using the SFFV API. Latency\n",
+        "increases, but should be roughly 100ms-200ms. We can still improve this by changing the grouping of keys."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "2a2f6dfd83f1"
+      },
+      "outputs": [],
+      "source": [
+        "%%timeit\n",
+        "sffv(\n",
+        "    data_client=data_client,\n",
+        "    feature_view=fv.resource_name,\n",
+        "    keys_list=[[f\"{num}\" for num in range(1, 1001)]],\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "e3dcc9bb38c0"
+      },
+      "source": [
+        "##### Changing grouping of keys (batch size tuning)\n",
+        "\n",
+        "In the `keys_list` argument to `sffv()`, we can change the grouping of the keys. This will affect how keys are read by the Feature Store server.\n",
+        "\n",
+        "The below `keys_list` will tell the Feature Store server to perform two reads, the first read will read entity 1-500 and the second read will read entity 501-1000. Placing contiguous entities in the same read will help reduce latency."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "baab4720ea2b"
+      },
+      "outputs": [],
+      "source": [
+        "%%timeit\n",
+        "sffv(\n",
+        "    data_client=data_client,\n",
+        "    feature_view=fv.resource_name,\n",
+        "    keys_list=[\n",
+        "        [f\"{num}\" for num in range(1, 501)],\n",
+        "        [f\"{num}\" for num in range(501, 1001)],\n",
+        "    ],\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "119fc46b49b5"
+      },
+      "source": [
+        "You can try your own \"batch size\" to see what works best."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "6d587dde7c0f"
+      },
+      "outputs": [],
+      "source": [
+        "%%timeit\n",
+        "batch_size = 50  # @param {type:\"number\"}\n",
+        "sffv(\n",
+        "    data_client=data_client,\n",
+        "    feature_view=fv.resource_name,\n",
+        "    keys_list=[\n",
+        "        [f\"{num}\" for num in range(i, i + batch_size + 1)]\n",
+        "        for i in range(1, 1001, batch_size)\n",
+        "    ],\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "235f1ada156d"
+      },
+      "source": [
+        "#### Using StreamingFetchFeatureValues with REST\n",
+        "\n",
+        "Below is an example of using the StreamingFetchFeatureValues API via REST\n",
```

Comment:
Suggested: "The following is an example of using the `StreamingFetchFeatureValues` API with REST."